frontend/src/embedding-sdk-bundle/components/public/dashboard/EditableDashboard/enterprise.unit.spec.tsx

```diff
@@ -10,26 +10,29 @@ import { screen, waitFor, within } from "__support__/ui";
 import { ROOT_COLLECTION } from "metabase/entities/collections";
```

Comment:
I could have tried to combine all the tests into a single spec, [like here](https://metaboat.slack.com/archives/C505ZNNH4/p1764622371438349). But this was already done.